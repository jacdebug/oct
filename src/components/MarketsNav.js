import React from 'react';
import styled from 'styled-components';
import MarketsNavLinks from './MarketsNavLinks';

const MarketsNav = ({ ...props }) =>
  <MarketsNavWrapper>
    {props.market.marketData
      ? <MarketsNavLinks {...props.market} />
      : <div>Loading... </div>}
  </MarketsNavWrapper>;

export default MarketsNav;

const MarketsNavWrapper = styled.div`
  align-self: flex-start;
  margin: 0 10px 0 0;
`;
