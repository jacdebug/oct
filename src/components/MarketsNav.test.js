import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MarketsNav from './MarketsNav';

const marketTrueValue = { marketData: {} };
const marketFalseValue = {};

it('renders without crashing', () => {
  shallow(<MarketsNav market={marketTrueValue} />);
  shallow(<MarketsNav market={marketFalseValue} />);
});

it('renders a snapshot', () => {
  const tree = shallow(<MarketsNav market={marketTrueValue} />);
  expect(toJson(tree)).toMatchSnapshot();
});
