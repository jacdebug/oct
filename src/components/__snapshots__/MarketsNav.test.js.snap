// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<styled.div>
  <MarketsNavLinks
    marketData={Object {}}
  />
</styled.div>
`;
